crates/forge-runner/src/gas.rs

```diff
@@ -106,3 +107,25 @@ fn get_onchain_data_segment_length(state_changes_count: StateChangesCount) -> us
 
     onchain_data_segment_length
 }
+
+pub fn check_available_gas(
+    available_gas: &Option<usize>,
+    summary: TestCaseSummary<Single>,
+) -> TestCaseSummary<Single> {
+    match summary {
+        TestCaseSummary::Passed {
+            name,
+            arguments,
+            gas_info,
+            ..
+        } if available_gas.map_or(false, |available_gas| gas_info > available_gas as u128) => {
+            TestCaseSummary::Failed {
+                name,
+                msg: Some("\n\tTest cost exceeded the available gas".to_string()),
```

Comment:
Will the actual gas be still displayed somewhere?

Reply:
Nope 🤔 I think I should add this